GitCommands/CommitDataManager.cs

```diff
@@ -194,7 +194,7 @@ private bool TryGetCommitLog([NotNull] string commitId, [NotNull] string format,
                 return false;
             }
 
-            var arguments = $"log -1 --pretty=\"format:{format}\" {commitId}";
+            var arguments = $"log --no-show-signature -1 --pretty=\"format:{format}\" {commitId}";
```

Comment:
❓ What's the first version of git that accepted this argument?

Reply:
➡️ Git added this for 2.10.0, as mentioned in the issue for this pull request.

Reply:
@gerhardol we'll need to bump the min version to 2.10

Reply:
I'd like to avoid patching 2.51 unless absolutely necessary